tests/components/abode/test_config_flow.py

```diff
@@ -141,46 +136,52 @@ async def test_step_mfa(hass: HomeAssistant) -> None:
             result["flow_id"], user_input={"mfa_code": "123456"}
         )
 
-        assert result["errors"] == {"base": "invalid_mfa_code"}
+    assert result["type"] is FlowResultType.FORM
+    assert result["step_id"] == "mfa"
+    assert result["errors"] == {"base": "invalid_mfa_code"}
 
     with patch("homeassistant.components.abode.config_flow.Abode"):
         result = await hass.config_entries.flow.async_configure(
             result["flow_id"], user_input={"mfa_code": "123456"}
         )
 
-        assert result["type"] is FlowResultType.CREATE_ENTRY
-        assert result["title"] == "user@email.com"
-        assert result["data"] == {
-            CONF_USERNAME: "user@email.com",
-            CONF_PASSWORD: "password",
-            CONF_POLLING: False,
-        }
+    assert result["type"] is FlowResultType.CREATE_ENTRY
+    assert result["title"] == "user@email.com"
+    assert result["data"] == {
+        CONF_USERNAME: "user@email.com",
+        CONF_PASSWORD: "password",
+        CONF_POLLING: False,
+    }
 
 
 async def test_step_reauth(hass: HomeAssistant) -> None:
     """Test the reauth flow."""
-    conf = {CONF_USERNAME: "user@email.com", CONF_PASSWORD: "password"}
-
     entry = MockConfigEntry(
         domain=DOMAIN,
         unique_id="user@email.com",
-        data=conf,
+        data={CONF_USERNAME: "user@email.com", CONF_PASSWORD: "password"},
     )
     entry.add_to_hass(hass)
 
-    with patch("homeassistant.components.abode.config_flow.Abode"):
-        result = await entry.start_reauth_flow(hass)
+    result = await entry.start_reauth_flow(hass)
 
-        assert result["type"] is FlowResultType.FORM
-        assert result["step_id"] == "reauth_confirm"
+    assert result["type"] is FlowResultType.FORM
+    assert result["step_id"] == "reauth_confirm"
 
-        with patch("homeassistant.config_entries.ConfigEntries.async_reload"):
-            result = await hass.config_entries.flow.async_configure(
-                result["flow_id"],
-                user_input=conf,
-            )
+    with (
+        patch("homeassistant.components.abode.config_flow.Abode"),
+        patch("homeassistant.config_entries.ConfigEntries.async_reload"),
```

Comment:
This seems unnecessary as we already use `mock_setup_entry` fixture to not load?

Reply:
> This seems unnecessary as we already use `mock_setup_entry` fixture to not load?

Removed